sdk/resourcemanager/Azure.ResourceManager/samples/DocSamples/Readme.cs

```diff
@@ -24,6 +24,26 @@ public void ClientAuth()
             ArmClient client = new ArmClient(new DefaultAzureCredential());
             #endregion Snippet:Readme_AuthClient
         }
+        [Test]
+        [Ignore("Only verifying that the sample builds")]
+        public void ClientAuth_China()
+        {
+            #region Snippet:Readme_AuthClient_China
+            // Please replace the following placeholders with your Azure information
+            string tenantId = "your-tenant-id";
+            string clientId = "your-client-id";
+            string clientSecret = "your-client-secret";
+            string subscriptionId = "your-subscription-id";
+            //ArmClientOptions to set the Azure China environment
+            var armOptions = new ArmClientOptions { Environment = ArmEnvironment.AzureChina };
+            // AzureAuthorityHosts to set the Azure China environment
+            var authorityHost = AzureAuthorityHosts.AzureChina;
+            // Create ClientSecretCredential for authentication
+            var credential = new ClientSecretCredential(tenantId, clientId, clientSecret, new TokenCredentialOptions { AuthorityHost = authorityHost });
```

Comment:
in general, in sample codes, we should avoid using `var`. Could you change these `var` here to their explicit type?
The type of `credential` here could be `TokenCredential`.